frontend/src/components/Forms/TeamEventCreditsForm/TeamEventsCreditDasboard.tsx

```diff
@@ -42,20 +33,19 @@ const TeamEventCreditDashboard = (props: { teamEvents: TeamEvent[] }): JSX.Eleme
         <h1>Check Team Event Credits</h1>
         <p>
           Check your team event credit status for this semester here! Every DTI member must complete
-          3 team event credits to fulfill this requirement.
+          {REQUIRED_TEC_CREDITS} team event credits to fulfill this requirement.
```

Comment:
```suggestion
           {REQUIRED_TEC_CREDITS} team event credits to fulfill this requirement.
```
Just a missing space here
<img width="259" alt="image" src="https://user-images.githubusercontent.com/81320443/201203245-1e027aa0-0d06-4846-99b9-838b4109e1c3.png">

Reply:
my suggestion didnt work but the space is still missing

Reply:
@henry-li-06 ^^^ the space still needs to be added btw